android/src/main/java/com/ledger/reactnative/RCTCoreBitcoinLikeTransaction.java

```diff
@@ -294,6 +295,7 @@ public void getTime(ReadableMap currentInstance, Promise promise) {
             Date javaResult = currentInstanceObj.getTime();
             WritableNativeMap result = new WritableNativeMap();
             DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
Can we create this object once in constructor?

Reply:
I guess we can, yes, but we should open a ticket to do so (I’d be happy to do it later) as it’s not on the path of this patch.